pt_os_web_portal/__main__.py

```diff
@@ -12,6 +12,8 @@
 logger = logging.getLogger()
 click_logging.basic_config(logger)
 
+logging.getLogger("pitop").setLevel(logging.INFO)
```

Comment:
for my own interest, would this not be the default logging level for the pitop module, and why are you changing it?

Reply:
this will make it so that only `INFO` level (and higher) messages from the `pitop` library are logged in the web-portal logs... by default, it prints from `DEBUG` level, and we definitely don't want that.... maybe there's another solution?